### ISSUE TITLE
how to use model?

### ISSUE THREAD

**Issue by nss2024** [ISU-PAAL/DeepDFA#11]
i trained DeepDFA + LineVul 
and finally create model.bin and mode.pt files amount (1GB)

and how to load model and how to run?
here is my code
====================================================================
# 모델 인스턴스를 생성한 후 가중치 로드
from transformers import RobertaForSequenceClassification, RobertaConfig
from linevul_model import Model
from transformers import RobertaForSequenceClassification, RobertaTokenizer, RobertaConfig
from code_gnn.models.flow_gnn.ggnn import FlowGNNGGNNModule  # FlowGNNGGNNModule 임포트
import argparse
config = RobertaConfig.from_pretrained("microsoft/codebert-base")
tokenizer = RobertaTokenizer.from_pretrained("microsoft/codebert-base")
config.num_labels =1
encoder = RobertaForSequenceClassification.from_pretrained("microsoft/codebert-base", config=config)

# flowgnn_encoder 설정
feat = "_ABS_DATAFLOW_datatype"  # 사용 가능한 임베딩 피처
input_dim = 1002  # 노드 특징 차원
hidden_dim = 32  # GNN 은닉층 크기
n_steps = 5  # GNN 스텝 수
num_output_layers = 3  # 출력 레이어 수
label_style = "graph"  # 레이블 스타일
concat_all_absdf = True  # 데이터 합치기 여부
encoder_mode = False  # encoder 모드 설정

# flowgnn_encoder 초기화
flowgnn_encoder = FlowGNNGGNNModule(
    feat=feat,
    input_dim=input_dim,
    hidden_dim=hidden_dim,
    n_steps=n_steps,
    num_output_layers=num_output_layers,
    label_style=label_style,
    concat_all_absdf=concat_all_absdf,
    encoder_mode=encoder_mode
)

#    do_train=True,
# do_test=True,
# train_data_file="/content/drive/MyDrive/train.csv",
# eval_data_file="/content/drive/MyDrive/val.csv",
# test_data_file="/content/drive/MyDrive/test.csv",
args = argparse.Namespace(
    model_name="model3.pt",
    output_dir="/content",
    model_type="roberta",
    tokenizer_name="microsoft/codebert-base",
    model_name_or_path="microsoft/codebert-base",
    epochs=1,
    block_size=512,
    train_batch_size=16,
    eval_batch_size=16,
    learning_rate=2e-5,
    max_grad_norm=1.0,
    evaluate_during_training=True,
    no_flowgnn=False,
    seed=3
)

model = Model(
    encoder=encoder,
    flowgnn_encoder=flowgnn_encoder,
    config=config,
    tokenizer=tokenizer,
    args=args
)

# 저장된 가중치 로드
# print(model)

model.load_state_dict(torch.load("model3.pt"),strict=False)
# model.load_state_dict(torch.load("model_line.bin"), strict=False)
